tests/ui/needless_continue.rs

```diff
@@ -49,6 +49,9 @@ fn main() {
 
         println!("bleh");
     }
+    loop {
+        continue;
+    }
```

Comment:
What about the more general case

````rust
loop {
    some_other_stmt();
    continue;
}
```

So, why not lint on every loop where ``continue`` is the last stmt/expr?

Reply:
I covered the issue mentioned in linked issue first, but that's a good point. I expanded it now to cover that case as well.